hack/aks/property-based-scheduling.md

```diff
@@ -0,0 +1,428 @@
+
+```sh
+# Similarly, create a namespace and deployment as placement template.
+kubectl create ns work-2
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: work-2-envelope
+  namespace: work-2
+  annotations:
+    kubernetes-fleet.io/envelope-configmap: "true"
+data:
+  app.yaml: |
+    apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      name: app
+      namespace: work-2
+    spec:
+      replicas: 1
+      selector:
+        matchLabels:
+          app: nginx
+      template:
+        metadata:
+          labels:
+            app: nginx
+        spec:
+            containers:
+            - name: nginx
+              image: nginx
+              ports:
+              - containerPort: 80
+              resources:
+                requests:
+                  cpu: 1
+                  memory: 2Gi
+EOF
+
+# Next, use the Fleet CRP API to place the workload.
+#
+# The workload will go to clusters with at least 16GB of memory.
+cat <<EOF | kubectl apply -f -
+apiVersion: placement.kubernetes-fleet.io/v1beta1
+kind: ClusterResourcePlacement
+metadata:
+  name: work-2
+spec:
+  resourceSelectors:
+    - group: ""
+      kind: Namespace
+      name: work-2
+      version: v1
+  policy:
+    # PickAll instructs Fleet to select all clusters if it matches with the specified
+    # affinity term.
+    placementType: PickAll
+    affinity:
+      clusterAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          clusterSelectorTerms:
+          - propertySelector:
+              matchExpressions:
+              - name: resources.kubernetes-fleet.io/available-memory
+                operator: Ge
+                values:
+                - "16Gi"
+EOF
+```
+
+The CRP API enables great flexibility; you can set up different requirements/preferences in combination, such as finding all clusters with at least 5 nodes and 10 available CPU cores, or 4 of all the clusters with the cheapest memory cost and the most amount of available memory. [Read Fleet's API definition to learn more](https://github.com/Azure/fleet/blob/main/apis/placement/v1beta1/clusterresourceplacement_types.go).
+
+## Clean things up
+
+To remove the resources you have created in this tutorial, simply delete the resource group with the command below:
+
+```sh
+az group delete -n $RG
+```
+
+All the AKS clusters in the resource group will be removed.
+
+## What's next
+
+Congratulations! You have completed the tutorial; we hope that you will appreciate the property-based scheduling capabilities we have added to Fleet. If you have any questions, feedback, or concerns, please raise [a GitHub issue](https://github.com/Azure/fleet/issues).
+
+To learn more about Fleet, see the [Fleet GitHub repository](https://github.com/Azure/fleet) and the [Fleet documentation](https://github.com/Azure/fleet/tree/main/docs).
```

Comment:
Consider linking directly to another Fleet feature tutorial that the user can test out, starting to create a "path" instead of having them decide what to do once they reach the broader Fleet repo.

Reply:
Done.